app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -53,6 +57,10 @@ protected void onCreate(Bundle savedInstanceState) {
         swipeableInfoCard = BottomSheetBehavior.from(infoCard);
 
         locationFragment = (LocationFragment) getSupportFragmentManager().findFragmentById(R.id.locationFragment);
+
+        AppDatabase appDB = AppDatabase.getInstance(this);
+        List<Shuttle> jas = appDB.shuttleDao().getScheduleByCampusAndDay("SGW", "Friday");
```

Comment:
What is the purpose of this here ? Was it only for testing? if yes, can you delete it ? and if no, I don't think this a  good place for it. Also, we should use proper variable names.

Reply:
Yeah, it was only for testing. I'll remove it. Good catch!